internal/tests/testrepository/header_testrepository.go

```diff
@@ -238,6 +238,39 @@ func (r *HeaderTestRepository) GetMerkleRootsConfirmations(
 	return mrcfs, nil
 }
 
+func (r *HeaderTestRepository) GetHeadersStartHeight(hashtable []string) (int, error) {
+	for i := len(*r.db) - 1; i >= 0; i-- {
+		header := (*r.db)[i]
+		for j := len(hashtable) - 1; j >= 0; j-- {
+			if header.Hash.String() == hashtable[j] && header.State == domains.LongestChain {
+				return int(header.Height), nil
+			}
+		}
+	}
+	return 0, nil
```

Comment:
Just as a reminder, if you change a logic in a real sql, you should change it here also.